docs/models/advanced/ibakedmodel.md

```diff
@@ -50,6 +50,47 @@ The `Random` parameter is used for randomised models.
 
 This method provides the [`IModelData`][modeldata] instance to the `getQuads` method and has access to the `ILightReader` world, `BlockPos` for the block which is being rendered and the [`IModelData`][modeldata] instance as passed from the `TileEntity`. By default this will pass the [`IModelData`][modeldata] parameter straight to the `getQuads` method but can be used to modify `ModelProperty`'s (see [Model Data][modeldata]). An example use case of this method is to provide the `BlockState` to render for a camouflage block from the neighbouring blocks.
 
+`BakedQuad`s
+============
+
+The `getQuads` method requires a list of quads to be drawn. These quads can either be generated algorithmically or can be copies of other block models using the `ModelManager`.
+
+!!! note
+When using the quads from another block model, if you are going to manipulate the quads, be sure to copy the quads to not override the quads from the original model.
```

Comment:
Admonition content needs to be indented: https://squidfunk.github.io/mkdocs-material/extensions/admonition/

Reply:
Thanks again for pointing this out. Should be fixed in the latest commit :)